fat/fat2/transactionbatch.go

```diff
@@ -5,22 +5,36 @@ import (
 	"fmt"
 
 	"github.com/Factom-Asset-Tokens/factom"
-	"github.com/Factom-Asset-Tokens/fatd/fat"
-	"github.com/Factom-Asset-Tokens/fatd/fat/jsonlen"
+	"github.com/Factom-Asset-Tokens/factom/fat103"
+	"github.com/Factom-Asset-Tokens/factom/jsonlen"
 )
 
 // TransactionBatch represents a fat2 entry, which can be a list of one or more
 // transactions to be executed in order
 type TransactionBatch struct {
 	Version      uint          `json:"version"`
 	Transactions []Transaction `json:"transactions"`
-	fat.Entry
+
+	Metadata json.RawMessage `json:"metadata,omitempty"`
+	Entry    factom.Entry    `json:"-"`
```

Comment:
Adam keeps the entry inside the other fat token transactions: https://github.com/Factom-Asset-Tokens/factom/blob/develop/fat1/transaction.go#L40-L48

I figured it might have to do with preserving the exact extids and content? Since json can be re-ordered. 

But we should just be consistent with what Adam has.